graphql/schema/types/config.graphql

```diff
@@ -17,6 +17,15 @@ type SpruceConfig {
   ui: UIConfig!
 }
 
+type AdminSettings {
+  announcements: Announcements
```

Comment:
The APIAdminSettings model doesn't have an `announcements` field in the shape of `Announcements` and is why you see a generated resolver in `config_resolver.go`. You can use [Banner and BannerTheme](https://github.com/evergreen-ci/evergreen/blob/64f77808e787c8522a51c647fdd996afd0de1a4b/rest/model/admin.go#L63-L64) instead and is what [SpruceConfig](https://github.com/evergreen-ci/evergreen/blob/64f77808e787c8522a51c647fdd996afd0de1a4b/graphql/schema/types/config.graphql#L7-L8) utilizes.

Reply:
makes sense thank you